File: app/src/main/java/com/scatl/uestcbbs/module/home/view/CollectionListView.java
package com.scatl.uestcbbs.module.home.view;

import com.scatl.uestcbbs.entity.CollectionListBean;

import java.util.List;

/**
 * author: sca_tl
 * date: 2020/5/3 11:08
 * description:
 */
public interface CollectionListView {
    void onGetCollectionListSuccess(List<CollectionListBean> collectionListBeans, boolean hasNext);
    void onGetCollectionListError(String msg);
}

File: app/src/main/java/com/scatl/uestcbbs/module/collection/view/CollectionListView.kt
package com.scatl.uestcbbs.module.collection.view

import com.scatl.uestcbbs.base.BaseView
import com.scatl.uestcbbs.entity.CollectionListBean

/**
 * Created by sca_tl at 2023/5/5 11:43
 */
interface CollectionListView: BaseView {
    fun onGetCollectionListSuccess(collectionListBeans: List<CollectionListBean>, hasNext: Boolean)
    fun onGetCollectionListError(msg: String?)
}